blivet/formats/disklabel.py

```diff
@@ -220,12 +220,13 @@ def parted_device(self):
 
     @classmethod
     def get_platform_label_types(cls):
-        label_types = ["msdos", "gpt"]
+        # always prefer gpt except for configurations below
+        label_types = ["gpt", "msdos"]
         if arch.is_pmac():
             label_types = ["mac"]
-        # always prefer gpt on aarch64, x86_64, and EFI plats except 32-bit ARM
-        elif arch.is_aarch64() or arch.is_x86(bits=64) or (arch.is_efi() and not arch.is_arm()):
-            label_types = ["gpt", "msdos"]
+        # prefet msdos on 32-bit ARM
+        elif arch.is_arm():
```

Comment:
It's confusing to me that `is_arm` means 32 bit, is this because the other arch string is `aarch64`?

Maybe worth a followup to rename `is_arm` to `is_arm32` or so

Reply:
The name is unfortunate, I had to check myself what `is_arm` actually means. We have `arch.is_arm` which means 32bit ARM and `arch.is_aarch64` for 64bit ARM.